docs/digger/installation/macos-prereqs.asciidoc

```diff
@@ -1,42 +1,128 @@
 [[macos-server]]
 === macOS Server pre-requisites
 
-NOTE: This is required only if you want to build iOS applications.
+This is required for building iOS applications. The following specifications are recommended:
 
-==== macOS and macOS Ansible Scripts Configuration
+* 4GB RAM
+* 2 Cores
+* A clean install of macOS 10.12 Sierra or cloud provider macincloud
```

Comment:
I think we might be removing the Sierra option? @finp can you confirm

Reply:
@laurafitzgerald  we're saying MacInCloud only for downstream, and that could change

Reply:
@finp ok thanks for clarification